code/modules/projectiles/projectile/magic.dm

```diff
@@ -200,10 +200,14 @@
 		if("animal")
 			var/path = pick(/mob/living/simple_animal/hostile/carp,
 							/mob/living/simple_animal/hostile/bear,
-							/mob/living/simple_animal/hostile/mushroom,
+							/mob/living/simple_animal/hostile/carp/eyeball,
+							/mob/living/simple_animal/hostile/eldritch/raw_prophet,
```

Comment:
evil

Reply:
I want more mobs that could end up hurting the wizard in the long run, a salty former crewmember turned 20 view range telepathic spy sounds like a good way to make wizards be a little more careful with polymorph